docs-site/content/guide/docsearch.md

```diff
@@ -109,6 +109,15 @@ The easiest way to run the scraper is using Docker.
     docker run -it --env-file=/path/to/your/.env -e "CONFIG=$(cat config.json | jq -r tostring)" typesense/docsearch-scraper:0.9.1
     ```
 
+  :::tip
+  If you are running Typesense on `localhost` and you're using Docker to run the scraper, you will need to change some things in your `config.json` file.
```

Comment:
Since this talks about the `config.json` file, could you move this tip under the `Create a DocSearch Scraper Config File` section?

Reply:
Of course! Already pushed a commit with the fix! Thanks!

Reply:
Thank you!